libs/moment-adapter/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@fundamental-ngx/moment-adapter",
+  "version": "VERSION_PLACEHOLDER",
+  "schematics": "./schematics/collection.json",
+  "description": "SAP Fundamentals, implemented in Angular",
+  "license": "Apache-2.0",
+  "homepage": "https://sap.github.io/fundamental-ngx/home",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/fundamental-ngx"
+  },
+  "scripts": {
+    "build": "../../node_modules/.bin/tsc -p tsconfig.schematics.json",
+    "copy:schemas": "cp schematics/*/schema.json ../../dist/libs/moment-adapter/",
+    "copy:collection": "cp schematics/collection.json ../../dist/libs/moment-adapter/schematics/collection.json",
+    "postbuild": "npm run && npm run copy:collection"
+  },
+  "peerDependencies": {
+    "@angular/common": "ANGULAR_VER_PLACEHOLDER",
```

Comment:
Should we add moment as a dependency for the adapter?

Reply:
added moment as dep